### PR TITLE
CORE-3640 - basic setup for modal experiment

### PR DESCRIPTION
**Pull request by amaciaszek-dsai**
This repository contains a script to run a project on modal cloud environment.

~~It will be slightly updated after PRs with config changes from @mariannaparzych are merged~~

### CODE REVIEW

**Comment by amaciaszek-dsai** [comment]
_train_from_recipe_modal.py_ script could be slightly cleaned further, but I didn't manage to do this in this week. It should be working fine in current setup

File: src/super_gradients/train_from_recipe_modal.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
+"""
```
[the remark this answers is not in the document]
Comment (mariannaparzych) [reply src/super_gradients/train_from_recipe_modal.py:1]: When I run it with above command I got:
```
Exception has occurred: TypeError       (note: full exception trace is shown but execution is paused at: _run_module_as_main)
main() missing 1 required positional argument: 'config_name'
```

File: README.md [hunk 2]
```diff
@@ -725,3 +725,29 @@ Features
...
+## README - unstructured.io sections
+
+### Modal
+
+#### Setup
+After you are added into _unstructured-io_ workspace on modal - run `modal setup` command to authorize and create token automatically.
+
+#### Run experiment
+To run training using modal you can run _train_from_recipe_modal.py_ script:
+`modal run src/super_gradients/train_from_recipe_modal.py --config-name train_unstructured_jan24_yolox`
```
Comment (amaciaszek-dsai) [inline README.md:740]: here ;) 
@mariannaparzych

**Review by mariannaparzych — APPROVED** [review]

File: src/super_gradients/train_from_recipe_modal.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+from super_gradients import Trainer, init_trainer
+
+dotenv.load_dotenv()
+
+stub = modal.Stub(name="train_from_recipe")
+
+# git branch and repository url for the image build
+# remember to have your changes pushed to the remote repository so that it can be accessed by the modal environment
+git_branch = "CORE-3640/modal_integration"  # set to None to use the default branch, remember to update it after merging
```
[the remark this answers is not in the document]
Comment (amaciaszek-dsai) [reply src/super_gradients/train_from_recipe_modal.py:20]: yes, I wanted to do it in a separate PR, but if you've finalized your tests and everything is working - I can also push the change now

[the remark this answers is not in the document]
Comment (amaciaszek-dsai) [reply src/super_gradients/train_from_recipe_modal.py]: changed to default

**Pull request merged**